apps/src/StudioApp.js

```diff
@@ -2707,7 +2708,8 @@ StudioApp.prototype.handleUsingBlockly_ = function(config) {
   // Never show unused blocks or disable autopopulate in edit mode
```

Comment:
Nit: can you update this comment now that we are sometimes disabling autopopulate?